ArraysHomework/ArraysHomework.cs

```diff
@@ -51,10 +63,13 @@ public void Arrays()
 
             // Swap the last and first value of this array.
             string[] swapMyValues = { "first", "middle", "last" };
-            
+            System.Array.Reverse(swapMyValues);
```

Comment:
`Reverse` only does what we want if there are exactly 2 or exactly 3 elements in the array. What if we didn't know what was in the array?